config/jobs/periodic/knative/operator/main/operator-main.gen.yaml

```diff
@@ -0,0 +1,67 @@
+periodics:
+  - name: knative-operator-main-periodic 
+    cluster: k8s-ppc64le-cluster
+    labels:
+      preset-knative-powervs: "true"
+    decorate: true
+    decoration_config:
+      gcs_configuration:
+        bucket: s3://ppc64le-prow-logs
+        path_strategy: explicit
+      s3_credentials_secret: s3-credentials
+    cron: "20 0 * * *"
+    extra_refs:
+      - base_ref: master
+        org: kantive
+        repo: operator
+        workdir: true
+    spec:
+      containers:
+        - image: quay.io/powercloud/knative-prow-tests:latest
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
+          command:
+            - runner.sh
+          args:
+            - bash 
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+
+              TIMESTAMP=$(date +%s)
+              K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/k8s-release-dev/ci/latest.txt)
+
+              kubetest2 tf --powervs-image-name CentOS9-Stream\
+                --powervs-region syd --powervs-zone syd05 \
+                --powervs-service-id af3e8574-29ea-41a2-a9c5-e88cba5c5858 \
+                --powervs-ssh-key knative-ssh-key \
+                --ssh-private-key ~/.ssh/ssh-key \
+                --build-version $K8S_BUILD_VERSION \
+                --cluster-name knative-$TIMESTAMP \
+                --workers-count 2 \
+                --up --auto-approve --retry-on-tf-failure 5 \
+                --break-kubetest-on-upfail true \
+                --powervs-memory 32
+          
+              export KUBECONFIG="$(pwd)/knative-$TIMESTAMP/kubeconfig"
+
+              git clone https://$(GITHUB_TOKEN)@github.ibm.com/ppc64le-automation/knative-upstream-ci.git 
```

Comment:
what are these scripts? right now in this repo, we are not dealing any of the code in github.ibm.com.